static/src/javascripts-legacy/projects/common/modules/atoms/story-questions.js

```diff
@@ -151,6 +223,25 @@ define([
                 bean.on(el, 'submit', submitSignUpForm);
             });
 
+            var answersDeliveryPreferences = $('.btn-answer-delivery-' + atomId);
+
+            answersDeliveryPreferences.each(function (el) {
+                bean.on(el, 'click', function(event) {
+                    submitDeliveryPreference(event)
+                    this.classList.add('is-clicked')
+                });
+            });
+
+            var finalCloseBtn = document.getElementById('js-final-thankyou-message-' + atomId);
+
+            bean.on(finalCloseBtn, 'click', function(event) {
```

Comment:
Here you can use `bean.once` I think (will automatically remove the listener)

Reply:
`one` - https://github.com/fat/bean#one